x-pack/plugins/reporting/export_types/printable_pdf/server/lib/layouts/print_layout.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import path from 'path';
+import { KbnServer, Size } from '../../../../../types';
+import { CaptureConfig } from './index.d';
+import { Layout } from './layout';
+
+type EvalArgs = any[];
+
+interface EvaluateOptions {
+  // 'fn' is a function in string form to avoid tslint from auto formatting it into a version not
+  // underfood by transform_fn safeWrap.
+  fn: ((...evalArgs: EvalArgs) => any) | string;
```

Comment:
we can actually get rid of the `| string` part now, since we switched back to a function.